sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -90,6 +113,16 @@ def test_series_tests(self):
     result = doctests.testmod(
         pd.core.series,
         use_beam=False,
+        report=True,
+        wont_implement={
```

Comment:
Maybe rename this to "wont_implement_ok"?

Reply:
Done!